sec_certs/dataset/dataset.py

```diff
@@ -82,7 +82,9 @@ def __getitem__(self, item: str):
         return self.certs.__getitem__(item.lower())
 
     def __setitem__(self, key: str, value: 'Certificate'):
-        dict(self.certs).update({key.lower(): value})
+        aux = dict(self.certs)
```

Comment:
I would rather keep the original code here (the one before the PR) and make mypy ignore this part of code all together, this is convoluted.